### PR TITLE
Fix missing docker dependencies

### PR DESCRIPTION
**Pull request by Putzzmunta**

### CODE REVIEW

**Comment by bghira** [comment]
this commit is labeled 'add WSL support' - does this inherently resolve the issues of running on Windows? if so, could you add a couple pointers on getting users started in `INSTALL.md` just under the "Installation" header to help some Windows users get a head start?

**Comment by Putzzmunta** [comment]
I only tested it with the docker engine within an ubuntu wsl2 distro. Therefore, I can't make a call if it is running on windows out of the box.
I will add a section in the docker guide, also explaining on how to setup a docker compose file.

File: INSTALL.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 For users that wish to make use of Docker or another container orchestration platform, see [this document](/documentation/DOCKER.md) first.
 
 ### Installation
+For  users operating on windows. An Installation guide based on Docker and WSL is found here [this document](/documentation/DOCKER.md).
```
Comment (bghira) [inline INSTALL.md]:
```suggestion

For  users operating on Windows 10 or newer, an installation guide based on Docker and WSL is available here [this document](/documentation/DOCKER.md).
```

File: documentation/DOCKER.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ This Docker configuration provides a comprehensive environment for running the S
 
 ## Getting Started
 
+### 0. Things to keep in mind when running on windows (WSL)
```
Comment (bghira) [inline documentation/DOCKER.md]:
```suggestion
### Windows OS support via WSL (Experimental)
```

File: documentation/DOCKER.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ This Docker configuration provides a comprehensive environment for running the S
 
 ## Getting Started
 
+### Windows OS support via WSL (Experimental)
+
+The following guide was tested in a WSL2 Distro that has the Dockerengine installed and  also contained the SimpleTuner repository.
```
Comment (bghira) [inline documentation/DOCKER.md]:
```suggestion
The following guide was tested in a WSL2 Distro that has Dockerengine installed.
```

File: documentation/DOCKER.md [hunk 4]
```diff
@@ -68,6 +73,43 @@ If you want to add custom startup scripts or modify configurations, extend the e
 
 If any capabilities cannot be achieved through this setup, please open a new issue.
 
+### Docker Compose
+If you want to use a `docker-compose.yaml` feel free to oriantate on the following template.
```
Comment (bghira) [inline documentation/DOCKER.md]:
```suggestion

For users who prefer `docker-compose.yaml`, this template is provided for you to extend and customise for your needs.
```

File: documentation/DOCKER.md [hunk 5]
```diff
@@ -68,6 +73,44 @@ If you want to add custom startup scripts or modify configurations, extend the e
...
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+```
+
+
```
Comment (bghira) [inline documentation/DOCKER.md]:
```suggestion
> ⚠️ Please be cautious of handling your WandB and Hugging Face tokens! It's advised not to commit them even to a private version-control repository to ensure they are not leaked. For production use-cases, key management storage is recommended, but out of scope for this guide.
```

**Comment by bghira** [comment]
thanks muchly. i have updated the documentation ( i hope you did not mind ) and will merge this now.

**Pull request merged**